test/Confluent.Kafka.IntegrationTests/Tests/AssignPastEnd.cs

```diff
@@ -23,12 +23,12 @@
 
 namespace Confluent.Kafka.IntegrationTests
 {
-    /// <summary>
-    ///     Test functionality of Consumer.Consume when assigned to offest
-    ///     higher than the offset of the last message on a partition.
-    /// </summary>
     public static partial class Tests
     {
+        /// <summary>
+        ///     Test functionality of Consumer.Consume when assigned to offest
+        ///     higher than the offset of the last message on a partition.
+        /// </summary>
```

Comment:
i tried setting auto.offset.reset to smallest and it does not change the behavior of the test.

note that by 'invalid offset' i mean 'out of range offset', not -1001.

it seems counter intuitive to me that auto.offset.reset would come into play for an explicitly specified offset, even if it's out of range.